### DISCUSSION TITLE
Have there been any proposals related to MoveTowardsAngle?

### DISCUSSION THREAD

**Discussion by Braveo** [godotengine/godot-proposals#5993] [Scripting]
I've been enjoying MoveTowards on velocities in order to give the player some movement acceleration, but I wish there was a MoveTowardsAngle so rotation can be smoothed easier. I know there's LerpAngle but I find it so weird that MoveTowardsAngle doesn't exist as it feels a lot more consistent to use with time. 

Easiest workaround is to just make my own functions but if LerpAngle exists then I feel like MoveTowardsAngle should exist as well.

**Comment by Zireael07** [comment]
Yes, there is already one https://github.com/godotengine/godot-proposals/issues/2074 and there's even an open PR.

Please search before starting a discussion..
